templates/security/csharp/securityImpl.csproj.in

```diff
@@ -52,7 +52,7 @@
   </PropertyGroup>
```

Comment:
This whole file should probably be replaced with a CMakeList.txt for language CSharp.